libsolidity/interface/CompilerStack.cpp

```diff
@@ -243,7 +243,8 @@ bool CompilerStack::parse()
 
 bool CompilerStack::analyze()
 {
-	if (m_stackState != ParsingSuccessful || m_stackState >= AnalysisSuccessful)
+	if ( (m_stackState != ParsingSuccessful && !m_parserErrorRecovery)
```

Comment:
Please take a look at our coding style or at the rest of the code.

Reply:
Sorry about that. I will try to be more attentive in the future. fafe88690 should address this.